apps/anoma_node/lib/node/transaction/ordering.ex

```diff
@@ -55,7 +64,7 @@ defmodule Anoma.Node.Transaction.Ordering do
     I am the type of the Ordering Enigine.
 
     I contain the Node for which the Ordering is launched, the upcoming
-    hight as well as a map from transaction IDs to their global order.
+    height as well as a map from transaction IDs to their global order.
```

Comment:
This should be a different PR. Please don't fix stuff in the same PR this leads to horrid conflicts that we must avoid